pkg/ignore/ignore.go

```diff
@@ -0,0 +1,127 @@
+package ignore
+
+import (
+	"bufio"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/openshift/source-to-image/pkg/api"
+)
+
+type DockerIgnorer struct {}
```

Comment:
I'm still not in favor of this naming, as I've pointed earlier, I'd call this just `Ignorer`, `DockerIgnorer` I'd reserve for the case where we'd actually support `.dockerignore` file, but it's up to you @bparees .